test/verify/test_layernorm.cpp

```diff
@@ -49,15 +49,17 @@ migraphx::instruction_ref add_layernorm(migraphx::module& m,
     auto pow            = m.add_instruction(migraphx::make_op("pow"), sub, exponent_mbcast);
     auto var            = m.add_instruction(migraphx::make_op("reduce_mean", {{"axes", {2}}}), pow);
     auto epsilon_mbcast = m.add_instruction(
-        migraphx::make_op("multibroadcast", {{"out_lens", {1, dims.at(1), 1}}}), epsilon);
+        migraphx::make_op("multibroadcast", {{"out_lens", {dims.at(0), dims.at(1), 1}}}), epsilon);
+
     auto add_epsilon = m.add_instruction(migraphx::make_op("add"), var, epsilon_mbcast);
     auto sqrt        = m.add_instruction(migraphx::make_op("sqrt"), add_epsilon);
     auto sqrt_mbcast =
         m.add_instruction(migraphx::make_op("multibroadcast", {{"out_lens", dims}}), sqrt);
     auto div = m.add_instruction(migraphx::make_op("div"), sub, sqrt_mbcast);
     auto scale_mbcast =
         m.add_instruction(migraphx::make_op("multibroadcast", {{"out_lens", dims}}), scale);
-    auto mul = m.add_instruction(migraphx::make_op("mul"), scale_mbcast, div);
+    auto mul = m.add_instruction(migraphx::make_op("mul"), div, scale_mbcast);
```

Comment:
I had to change order because matcher is only matching for the first arg 
https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/0974bc455096e0492d95f85710628839f2013331/src/targets/gpu/fuse_ops.cpp#L793

I think matcher should look for either arg but that requires some change inside `apply()`.

Reply:
@pfultz2  what do you think about this one ? Should i update the matcher ?

Reply:
Make that a separate PR/Change incase that breaks things.

Reply:
Yea I wouldnt worry about it in this PR.

Reply:
> Yea I wouldnt worry about it in this PR. 

i mean as a separate change/PR